scripts/tests/lib.sh

```diff
@@ -11,5 +11,5 @@ function remove_iptables_drop_rules() {
 }
 
 function client_ping_resource() {
```

Comment:
I can't find the "suggest change" button in Github.

If this is curl it should be `client_check_resource` or something, just looking at the name I would assume 'ping' is a literal ICMP ping.